x/slashing/client/rest/query.go

```diff
@@ -19,46 +18,98 @@ func registerQueryRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Co
 		signingInfoHandlerFn(cliCtx, slashing.StoreKey, cdc),
 	).Methods("GET")
 
+	r.HandleFunc(
+		"/slashing/signing_infos",
```

Comment:
Should this be `/slashing/validators/signing_infos`?

Reply:
It will collide with {validatorPubKey}, that was the initial thinking behind #3964

Reply:
So in this specific case it will work because we never request the resource 'validator'